Library/Homebrew/formula.rb

```diff
@@ -1548,6 +1548,14 @@ def runtime_dependencies
     runtime_dependencies + runtime_requirement_deps
   end
 
+  def runtime_requirements
```

Comment:
Let's just have this method in `formula_installer` until we need it for stuff in `formula`